packages/commonwealth/client/scripts/views/pages/CreateCommunity/steps/CommunityTypeStep/CommunityTypeStep.tsx

```diff
@@ -89,21 +90,35 @@ const CommunityTypeStep = ({
         a community. Members must also have a compatible wallet type to join
         your community.
       </CWText>
-
-      <CWCommunitySelector
-        key={ethereumOption.type}
-        type={ethereumOption.type}
-        title={ethereumOption.title}
-        description={ethereumOption.description}
-        isRecommended={ethereumOption.isRecommended}
-        onClick={() =>
-          handleCommunitySelection({
-            type: ethereumOption.type,
-            chainBase: ethereumOption.chainBase,
-          })
-        }
-      />
-
+      <div className="advanced-options-container">
```

Comment:
Why do we have two advanced-options-containers here?

Reply:
The class can be re used to style the two cards that we now have in the non-advanced option section. Otherwise there's no margin etc